### PR TITLE
Disable relationship validations

### PR DESCRIPTION
**Pull request by amitsuryavanshi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ Features:
 
 - Relationships mentioned in sparse field param (and not in include), will now be returned in relationship block of response
 
+## 0.1.20
+
+Features:
+
+- Disableing call save on reationships objects on a post/put request. Disabling sidepost. We can update relationships as earlier on post/put requests.
```
Comment (mrhardikjoshi) [inline CHANGELOG.md]:
```suggestion
- Disabling calling save on relationship objects on a post/put requests. Disabling sideposting. We can update relationships as earlier on post/put requests.
```

File: .github/workflows/specs.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: [ jruby, ruby ]
+        ruby-version: [ jruby-9.3.8.0, ruby ]
```
Comment (mrhardikjoshi) [inline .github/workflows/specs.yml]: I have freezed jruby version to `jruby-9.3.9.0` in master branch so you can remove this change and this branch will still pass the build.

File: lib/graphiti/active_graph/adapters/active_graph.rb [hunk 3]
```diff
@@ -147,8 +147,13 @@ def attributes_for_has_many(rel_attrs)
       end
 
       def process_relationship_attrs(x, rel_attrs, assign_multiple)
-        x[:object] = x[:resource]
-          .persist_with_relationships(x[:meta], x[:attributes], x[:relationships], self, x[:foreign_key])
+        x[:object] = find_record(x)
```
Comment (mrhardikjoshi) [inline lib/graphiti/active_graph/adapters/active_graph.rb:150]: have a new global config [here](https://github.com/graphiti-api/graphiti/blob/14e84ef7e10c347ec391b96948365b5195074632/lib/graphiti/configuration.rb#L3) to support sideposting optionally? default being sideposting enabled.

File: lib/graphiti/active_graph/adapters/active_graph.rb [hunk 4]
```diff
@@ -170,6 +168,16 @@ def resource_association_value(rel_map)
           rel_map[:object]
         end
       end
+
+      def find_record(x)
+        if Graphiti.config.respond_to?(:allow_sidepost) && !Graphiti.config.allow_sidepost
```
Comment (mrhardikjoshi) [inline lib/graphiti/active_graph/adapters/active_graph.rb]: Once we have `attr_accessor :allow_sidepost` here in this gem, will we need this check `Graphiti.config.respond_to?(:allow_sidepost)`? I think we will be able to access the config without checking with `respond_to`

Comment (amitsuryavanshi) [reply lib/graphiti/active_graph/adapters/active_graph.rb]: makes sense

**Review by mrhardikjoshi — APPROVED** [review]

**Pull request merged**